demo/kubernetes/rook-operator.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: rook-operator
+  namespace: rook
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: rook-operator
+    spec:
+      containers:
+      - name: rook-operator
+        image: quay.io/rook/rook-operator:dev-2017-02-01-c
+        command: ["/opt/rook/bin/rook-operator"]
```

Comment:
You probably dont need to specify a command arg since it is the same as the `EntryPoint` in the dockerfile.